webpack/webpack.config.base.js

```diff
@@ -135,6 +135,7 @@ const config = {
       {
         test: /\.(ts|tsx)$/,
         exclude: /node_modules/,
+        // #4164 TODO investigate removing babel
```

Comment:
@kyranjamie - I had meant to tag you here. I made a new issue for this https://github.com/leather-wallet/extension/issues/4199 

Do you think removing `babel` is something you could do easily? I started to try but I was un-sure and had some difficulty with it